addons/portal/views/portal_templates.xml

```diff
@@ -129,11 +129,11 @@
                 </a>
                 <div t-attf-class="dropdown-menu js_usermenu #{_dropdown_menu_class}" role="menu">
                     <a groups="base.group_user" href="/odoo" role="menuitem" class="dropdown-item ps-3" id="o_backend_user_dropdown_link">
-                        <i class="fa fa-fw fa-th me-1 small text-primary"/> Apps
+                        <i class="fa fa-fw fa-th me-1 small text-primary-emphasis"/> Apps
```

Comment:
@stefanorigano I ideally should not remove class from XML templates in stable. Could you check if this PR work by adding the emphasis class but keeping the non-emphasis class at the same time? (then in master, we would remove the old one).

Reply:
> Could you check if this PR work by adding the emphasis class but keeping the non-emphasis class at the same time?

It doesn't out of the box. The problem is that "our" `text-primary` takes priority over default bootstrap classes, since it's defined in [bootstrap_review_frontend](https://github.com/odoo/odoo/blob/18.0/addons/web/static/src/scss/bootstrap_review_frontend.scss#L14) .
I have a workaround tho, I'll ping you when done 👍